Terminal.Gui/Windows/Wizard.cs

```diff
@@ -203,11 +203,6 @@ public WizardStep (ustring title)
 
 				base.Add (contentView);
 
-				helpTextView.ColorScheme = new ColorScheme () {  
```

Comment:
Why is this change part of this PR?

Reply:
Because it only have the appropriate effect with this PR.

Reply:
Yes it does, because the Wizard use the `TextView` and you changed the `ColorScheme` to looks better with the old `TextView` `ColorScheme`. With this PR, the changes some of their colors will affect how the `Wizards` scenario will display with this PR.

Reply:
Please answer this question: 

Did you make these changes to fix Wizard, or to fix a broader problem in TextView and TextField?

Reply:
@tig I did to fix `Wizard` after fixing a broader problem in `TextView` and `TextField`.

Reply:
> @tig I did to fix `Wizard` after fixing a broader problem in `TextView` and `TextField`.

I'm still not getting my thoughts to you clearly. My apologies.

I do not think this PR should include changes to Wizard. I think it should be two PRs, fixing two Issues:

1) TextView/TextField use different colors.
2) Wizard no longer needs to set specific colors, because 1) has been fixed.

Reply:
> 2\. Wizard no longer needs to set specific colors, because 1) has been fixed.

Done #2012 teacher @tig :-)